functions/src/api/v1/reminders/deleteReminder.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from 'express'
+import { wrap } from 'async-middleware'
+import { assertRightToEditRoom } from '../../../db/assertRightsToEditRoom'
+import { deleteReminderDb } from '../../../db/remindersDb'
```

Comment:
I thought about using a repository to manage this but it does not make sense as we are not storing any values to later on access. Also, using a Reminder class like new Reminder() will add extra db request to get the Reminder from db before deleting it. 
So maybe we could have ReminderDb with statis classes, or do you propose anything else?

Reply:
https://github.com/Instant-Visio/InstantVisio-WebApp/issues/348

Reply:
I would stick with the `Reminder` class, implement a static method within it to delete a `reminder` without getting it, but still have some methods that allow getting the reminder, and do something with that (in case those are used somewhere else)

Reply:
but a Reminder is an object, it's weird to use it has static object, anyway, let's discuss this in the issue